src/core/Microsoft.PowerToys.Settings.UI.Runner/MainWindow.xaml.cs

```diff
@@ -25,13 +25,9 @@ private void WindowsXamlHost_ChildChanged(object sender, EventArgs e)
 
             if (shellPage != null)
             {
-                shellPage.SetRestartElevatedCallback(delegate(string msg) 
+                // set restart as admin configuration and restart.
+                shellPage.SetRestartElevatedCallback(delegate (string msg)
```

Comment:
The `SetRestartElevatedCallback` implementation may have to be revisited, since it should not do anything special beside sending a message to the runner. The runner will send a quit event to the settings.

Reply:
Fixed it here: https://github.com/microsoft/PowerToys/pull/1702/commits/94196b9d7eb161e98c10185c781ea1b7d25e775d 

Currently, I have not implemented functionality to close the app when the runner closes. This implementation was intended to close the application when the runner closes but since all the operations will be done in the runner I guess there is no need to start the settings application as admin? :) Thanks.